atlasdb-jdbc/src/main/java/com/palantir/atlasdb/keyvalue/jdbc/JdbcAtlasDbFactory.java

```diff
@@ -38,7 +38,15 @@ public String getType() {
         return JdbcKeyValueConfiguration.TYPE;
     }
 
-    // async initialization not implemented/propagated
+    /**
+     * Creates a JdbcKeyValueService.
```

Comment:
Again, we could use a note explicitly calling out that async initialisation is not supported.